component/stream/stream.go

```diff
@@ -67,11 +68,16 @@ type StreamOptions struct {
 	Policy            string `config:"policy" yaml:"policy,omitempty"`
 	CacheOnFileHandle bool   `config:"cache-on-file-handle" yaml:"cache-on-file-handle,omitempty"`
 	readOnly          bool   `config:"read-only"`
+	Persistence       bool   `config:"persistence"`
```

Comment:
since all disk related options are prefixed with "disk" we should do this here as well so `disk-persistence`

Reply:
done